src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -378,7 +379,7 @@ export class AccountManagementService implements IAccountManagementService {
 		const updatedAccounts = await provider.initialize(accounts);
 
 		// Don't add the accounts that are about to get deleted to the cache.
-		this._providers[providerMetadata.id].accounts = updatedAccounts.filter(s => s.delete === false);
+		this._providers[providerMetadata.id].accounts = updatedAccounts.filter(s => s.delete === undefined);
```

Comment:
This should be `s => !s.delete` I would think - if delete is undefined OR false then we should be keeping it.

(it may be the case that it just happens to always be undefined or true right now, but we shouldn't assume that will always stay the case)